doc/website/getting-started/overview.md

```diff
@@ -302,346 +52,262 @@ else
+First we need to start RouDi.
 
-#### Receiving data
-For simplicity we assume that we periodically check for new data. It is also possible to explicitly wait for data
-using the [Waitset](../../../iceoryx_examples/waitset/README.md). The code to receive the data is the same, the only
-difference is the way we wake up before checking for data.
+```bash
+# If installed and available in PATH environment variable
+iox-roudi
+# If build from scratch with script in tools
+$ICEORYX_ROOT/build/iox-roudi
+```
 
-##### 1. Take and process a sample
+Afterwards, we can start the applications which immediately connect to the RouDi via their runtime.
 
-```cpp
-while (keepRunning)
-{
-    // wait for new data (either sleep and wake up periodically or by notification from the waitset)
+When the application terminates, the runtime cleans up all resources needed for communication with RouDi. This
+includes all memory chunks used for the data transmission which may still be held by the application.
 
-    auto result = subscriber->take();
+<!-- @todo add overview graphic -->
 
-    if(!result.has_error())
-    {
-        auto& maybeSample = result.value();
-        if (maybeSample.has_value())
-        {
-            auto& sample = maybeSample.value();
-            const CounterTopic* counter = sample.get();
-            //process the data
-
-            //alternatively use operator->
-            uint32_t counter = sample->counter;
-        }
-        else
-        {
-            // we received no data
-        }
-    } else {
-        iox::popo::ChunkReceiveResult& error = result.get_error();
-        //handle the error
-    }
-}
+We now briefly define the main entities of an iceoryx system which were partially already used in the example above.
 
-```
+### RouDi
 
-By calling ``take`` we get a ``iox::cxx::expected<iox::optional<iox::popo::Sample<const CounterTopic>>>``. Since this
-may fail, we have to handle a potential error. If there is no error, we may still have not recived a sample, indicated by
-a ``nullopt``. Otherwise we can get the pointer of our data from the sample and process the received data.
+RouDi is an abbreviation for **Rou**ting and **Di**scovery. RouDi takes care of the communication setup but does not
+actually participate in the communication between the publisher and the subscriber. RouDi can be thought of as the
+switchboard operator of iceoryx. One of its other major tasks is the setup of the shared memory, which the
+applications use for exchanging payload data. Sometimes referred to as daemon, RouDi manages the shared memory and is
+responsible for the service discovery, i.e. enabling subscribers to find topics offered by publishers. It also keeps
+track of all applications which have initialized a runtime and are hence able to use publishers or subscribers.
 
-We can shorten this somewhat by using a functional approach.
+When an application terminates, RouDi cleans up all resources needed for the communication with RouDi. If an
```

Comment:
we have to distinguish here between graceful and abnormal termination since there is the same sentence on line 117 `When the application terminates`